dmutils/logging.py

```diff
@@ -138,7 +138,7 @@ def format(self, record):
         try:
             record.msg = record.msg.format(**record.__dict__)
         except KeyError:
-            logger.error("failed to format log message")
+            logger.exception("failed to format log message")
```

Comment:
Yes, but it wouldn't add it to the message so it would be a little harder to spot immediately.